test/testlocal/nft/anvil/FakeBasicActions.sol

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: GPL-3.0
```

Comment:
Can you please add some documentation to understand why this file is included? Maybe a reference to the relevant issue

Reply:
added some information to the file, it was originally part of the PoC for the issue.